hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -256,6 +259,7 @@ public void testMetadataTableBootstrap(HoodieTableType tableType, boolean addRol
     validateMetadata(testTable, true);
   }
 
+  @Disabled("HUDI-9221")
```

Comment:
What's the reason of disabling these tests?  The tests seem to be unrelated to the changes in this PR?

Reply:
we have enabled it in https://github.com/apache/hudi/commit/07328c76019588da2b828e834847023f8621f9f7 
will re-enable and test it out

Reply:
Reenabled these tests